tests/test_kernels.py

```diff
@@ -255,16 +257,8 @@ def test_zero_area(vector_grid_type_data_and_extra_kwargs):
 
     grid_type, (data_u, data_v), extra_kwargs = vector_grid_type_data_and_extra_kwargs
 
-    test_kwargs = copy.deepcopy(extra_kwargs)
-    # fill area_u, area_v with zeros over land; e.g., you will find that in MOM6 model output
-    test_kwargs["area_u"] = np.where(
-        extra_kwargs["wet_mask_t"] > 0, test_kwargs["area_u"], 0
-    )
-    test_kwargs["area_v"] = np.where(
-        extra_kwargs["wet_mask_t"] > 0, test_kwargs["area_v"], 0
-    )
```

Comment:
@NoraLoose - I took this block and moved it into the fixture for the mom vector grid data. Do you see any problem with that?

Reply:
Yes, it fits better in the fixture for the mom vector grid data (where you have moved it). The solid body rotation test itself should not need `area = 0` over land.